### PR TITLE
Added new example form

### PR DESCRIPTION
**Pull request by joeribekker**

### CODE REVIEW

File: docs/manual/forms/examples/fields_for_logintype.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+Velden afhankelijk van inlogmethode
+===================================
+
+Er kunnen per formulier meerdere inlogmethoden worden geselecteerd. Afhankelijk
+van de inlogmethode kunnen andere velden worden getoond. Dit kan handig zijn
+voor formulieren die geschikt zijn voor bedrijven en voor particulieren.
+
+.. note::
+    
+    Open Formulieren ondersteund niet direct de logica om op basis van
```
Comment (sergei-maertens) [inline docs/manual/forms/examples/fields_for_logintype.rst:11]:
```suggestion
    Open Formulieren ondersteunt niet direct de logica om op basis van 
```

File: docs/manual/forms/examples/fields_for_logintype.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+voor formulieren die geschikt zijn voor bedrijven en voor particulieren.
+
+.. note::
+    
+    Open Formulieren ondersteund niet direct de logica om op basis van 
+    inlogmethode bepaalde velden wel of niet te tonen. We beschrijven hier een
+    methode om het toch voor elkaar te krijgen. Dit vereist wel dat er een 
+    koppeling met de :ref:`KvK <configuration_prefill_kvk>` en met 
+    :ref:`StUF-BG <configuration_prefill_stuf_bg>` of 
+    :ref:`Haal Centraal Bevragen personen <configuration_prefill_haal_centraal>`.
```
Comment (sergei-maertens) [inline docs/manual/forms/examples/fields_for_logintype.rst:16]:
```suggestion
    methode om het toch voor elkaar te krijgen. Dit vereist wel dat er een 
    koppeling met de :ref:`KvK <configuration_prefill_kvk>` en met 
    :ref:`StUF-BG <configuration_prefill_stuf_bg>` of 
    :ref:`Haal Centraal Bevragen personen <configuration_prefill_haal_centraal>`
    is ingesteld.
```

File: docs/manual/forms/examples/fields_for_logintype.rst [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+In dit voorbeeld gaan we er van uit dat u een
+:ref:`eenvoudig formulier <example_simple_form>` en combineren we de kennis van
+:ref:`voorinvullen <example_prefill>` met :ref:`logica <example_logic_rules>`.
```
Comment (sergei-maertens) [inline docs/manual/forms/examples/fields_for_logintype.rst:20]:
```suggestion
In dit voorbeeld gaan we er van uit dat u een
:ref:`eenvoudig formulier <example_simple_form>` heeft en combineren we de kennis van
:ref:`voorinvullen <example_prefill>` met :ref:`logica <example_logic_rules>`.
```

File: docs/manual/forms/examples/prefill.rst [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
-:ref:`KvK <configuration_prefill_kvk>` prefill kan gebruiken maar we hebben op dit moment ook
-:ref:`StUF-BG <configuration_prefill_stuf_bg>` prefill en
-:ref:`Haal Centraal Bevragen personen <configuration_prefill_haal_centraal>` prefill dat u kan gebruiken.
+.. _example_prefill:
+
+=========================
+Voorinvullen van gegevens
+=========================
+
+In dit voorbeeld tonen we hoe u velden kunt voorinvullen. In dit voorbeeld
```
Comment (sergei-maertens) [inline docs/manual/forms/examples/prefill.rst:7]: "In dit voorbeeld" start in both sentences is a bit weird

File: docs/manual/forms/examples/prefill.rst [hunk 5]
```diff
@@ -1,12 +1,18 @@
...
+.. _example_prefill:
+
+=========================
+Voorinvullen van gegevens
+=========================
+
+In dit voorbeeld tonen we hoe u velden kunt voorinvullen. In dit voorbeeld 
+tonen we hoe informatie uit de :ref:`KvK <configuration_prefill_kvk>` 
+vooringevuld kan worden.
+Het vooronvullen van gegevens kan uit diverse bronnen komen, naast die van de
```
Comment (sergei-maertens) [inline docs/manual/forms/examples/prefill.rst:10]:
```suggestion
Het voorinvullen van gegevens kan uit diverse bronnen komen, naast die van de 
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: docs/manual/forms/examples/prefill.rst (hunk 4 above)
Comment (joeribekker) [reply docs/manual/forms/examples/prefill.rst]: Iets herschreven.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**